### ISSUE TITLE
Cargo says 0.2.0 is the latest version

### ISSUE THREAD

**Issue by wheelerchristopherg** [str4d/age-plugin-yubikey#76]
## Environment

* OS: Arch linux
* age-plugin-yubikey version: 0.2.0

## What were you trying to do
Update to 0.3.0

## What happened
Cargo says 0.2.0 is the latest.

```
$ age-plugin-yubikey -V
age-plugin-yubikey 0.2.0

$ cargo install-update age-plugin-yubikey
    Updating registry 'https://github.com/rust-lang/crates.io-index'

Package             Installed  Latest  Needs update
age-plugin-yubikey  v0.2.0     v0.2.0  No

No packages need updating.
Overall updated 0 packages.
```

**Comment by mihaigalos** [comment]
Last crates.io published version is [`0.2.0`](https://crates.io/crates/age-plugin-yubikey).
The GH Actions don't seem to have an automatic deployment to `crates.io`, looks like @str4d is doing it manually.
Let me know if I can contribute a CD pipeline for cargo here.

**Comment by str4d** [comment]
Whoops, that was just me forgetting to run `cargo publish`. [Fixed!](https://crates.io/crates/age-plugin-yubikey/0.3.0)
